src/httpServer/handler.ts

```diff
@@ -46,7 +46,7 @@ export class HandlerContextImpl extends OperonContextImpl implements HandlerCont
       extractedSpanContext.isRemote = true;
       span = operon.tracer.startSpanWithContext(extractedSpanContext, koaContext.url, spanAttributes);
     }
-    super(koaContext.url, span, operon.logger);
+    super(koaContext.url, span, operon.logger, operon.config.applicationVersion);
```

Comment:
All context constructors pass the `applicationVersion` to the parent `OperonContext`